research/document_types/draft_classification.py

```diff
@@ -0,0 +1,169 @@
+import logging
+import pathlib
+import sys
+import dotenv
+import os
+import mlflow
+import numpy as np
+from sklearn.metrics import (
+    classification_report,
+    confusion_matrix,
+    precision_recall_fscore_support
+)
+from sklearn.model_selection import train_test_split, StratifiedKFold, cross_validate
+import pickle as pkl
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from tqdm import tqdm
+import toml
+
+from research.lib import data_access, embeddings
+
+REPOSITORY_ROOT = (pathlib.Path().cwd() / ".." / "..").resolve()
+sys.path.append(str(REPOSITORY_ROOT))
+
+dotenv.load_dotenv()
+
+with open('draft_classification.toml', 'r') as f:
+    config = toml.load(f)
```

Comment:
```suggestion
config = toml.load("draft_classification.toml")
```

However, it's also a side effect that would preferably happen in `if __name__ == "__main__"`...